.github/workflows/CI.yml

```diff
@@ -88,7 +100,20 @@ jobs:
 
       - name: Run build
         run: |
-          ./gradlew build
+          git clone https://github.com/NanXiao/lscpu.git
```

Comment:
Can you use `sysctl -a | grep cpu.feat | grep -i avx2`? Should make time faster

Reply:
Updated